packages/wdio-sauce-service/src/service.ts

```diff
@@ -247,6 +247,7 @@ export default class SauceService implements Services.ServiceInstance {
 
         const files = (await fs.promises.readdir(this._config.outputDir))
             .filter((file) => file.endsWith('.log'))
+        for (let file in files) { file.replace((/[[]\d\d\w-?\+?/g), '') }
```

Comment:
```suggestion
        for (let file in files) {
            file.replace(/\[\d\d\w-?\+?/g, '')
        }
```

Reply:
I wonder if there is a little NPM package for this which strips out color asciis like that.